src/main/java/com/iota/iri/hash/Curl.java

```diff
@@ -8,9 +8,8 @@
 
 /**
  * (c) 2016 Come-from-Beyond and Paul Handy
- *
+ * <p>
```

Comment:
No changes in this Curl.java class except paragraph symbol that the editor added - unbeknownst to me!